README.md

```diff
@@ -121,6 +121,24 @@ List<Tx> txs = client.getTxsByHeight(height);
 List<TxResponse> txResponses = client.getTxResponsesByHeight(height);
 ```
 
+### Txs and TxResponses with pagination
+```java
+int offset = 0, limit = 2, total = 0;
+while (true) {
+    PageRequest pagination = PageRequest.newBuilder().setOffset(offset).setLimit(limit).setCountTotal(true).build();
+    GetTxsEventResponse resp = client.getTxsByHeight(height, pagination);
+    
+    List<Tx> txs = resp.getTxsList();
+    List<TxResponse> txResponses = resp.getTxResponsesList();
+
+    offset += resp.getTxsCount();
+    total += resp.getTxsCount();
+    if (total == resp.getPagination().getTotal()) {
```

Comment:
How about chaning like below?
```suggestion
    if (total >= resp.getPagination().getTotal()) {
```

If the total count of query is changed, this loop can run infinititely.(as the total count decrease)